packages/renderer/src/lib/volume/CreateVolume.svelte

```
@@ -25,16 +26,43 @@ onMount(async () => {
+    .some(volume => volume.Name === nameValue);
+
+  if (volumeAlreadyExists) {
+    volumeNameError = `The name "${nameValue}" already exists. Please choose a different name.`;
+    invalidName = true;
+  } else {
+    volumeNameError = undefined;
+    invalidName = false;
+  }
+}
+
+$: checkVolumeName(volumeName, selectedProvider);
```

Biome (2.4.15)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)